### PR TITLE
Add directory input

### PR DESCRIPTION
**Pull request by viktigpetterr**
Hi there! 
How about something like this @thisismydesign? :blush: 
Relates to: #13

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -1,6 +1,10 @@
 name: 'Yarn install cache'
 author: 'c-hive'
 description: '1-liner yarn install cache for GitHub Actions'
+inputs:
+  directory:
+    description: 'Directory containing yarn.lock'
+    default: ''
```
Comment (thisismydesign) [inline action.yml]:
```suggestion
    default: './'
```

**Review by thisismydesign — COMMENTED** [review]
This is great, thanks @viktigpetterr! <3

Left 1 suggestion to make the default dir explicit.

**Comment by viktigpetterr** [comment]
No worries @thisismydesign :smile: 
Also that was good suggestion :+1:

**Comment by viktigpetterr** [comment]
I realized that b34096d would be more permissive when choosing directory :smile:  @thisismydesign

**Comment by thisismydesign** [comment]
Thanks! I'll be able to get this released probably next week.

**Comment by thisismydesign** [comment]
Thanks for your contribution @viktigpetterr and sorry for the delay! Released it as v2.1.0 and bumped the v2 tag so you can use this right away with your existing workflow.

**Pull request merged**